src/api/services/charactersService.ts

```diff
@@ -45,4 +54,20 @@ export namespace CharactersService {
     );
     return PaginationMapper.mapPaginationFromDto(data, CharacterMapper.fromDto).items[0];
   };
+
+  /**
+   * Gets a page of characters.
```

Comment:
missing `@param`, does eslint not work properly?

Reply:
Well yeah, no error